client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -945,10 +944,13 @@ export default {
     triggerStatementDeletion (id) {
       if (window.confirm(Translator.trans('check.statement.delete'))) {
         this.deleteStatement(id)
-          .then(response => checkResponse(response, {
-            200: { type: 'confirm', text: 'confirm.statement.deleted' },
-            204: { type: 'confirm', text: 'confirm.statement.deleted' }
-          }))
+          .then(response => {
+            this.getItemsByPage(this.pagination.currentPage)
+            checkResponse(response, {
+              200: { type: 'confirm', text: 'confirm.statement.deleted' },
+              204: { type: 'confirm', text: 'confirm.statement.deleted' }
+            })
+          })
```

Comment:
shouldn't the getItemsByPage be fired after the checkResponse?
what if the request fails. Is it on purpose, that the page gets reloaded then?

Reply:
of course, you are totally right (: fixed it